src/entities/Order.js

```diff
@@ -1,8 +1,13 @@
 export class Order {
-    constructor(id, clientId, dishes, totalPrice) {
+    constructor(id, clientId, orderDetails, totalPrice, status, chef, cashier, date) {
```

Comment:
Rename `date` to `teansactionDate`

Reply:
Renamed date property to transactionDate.